{{ cookiecutter.project_name }}/{{ cookiecutter.package_name }}/scripts/train.py

```diff
@@ -22,7 +22,16 @@
     "config_file", type=click.Path(exists=True), default="/mnt/configs/config.yml"
 )
 @setup_logging_env
-def main(config_file="config.yml"):
+def _main(config_file):
+    """
+    This pattern aids in writing tests, as the function with the click decorators can be
+    separately tested from the script functionality itself. In addition, the main()
+    function can now be imported and used in other python modules
+    """
+    main(config_file=config_file)
+
+
+def main(config_file):
```

Comment:
I tried a couple of different ways to do `_main.__doc__ = main.__doc__` but was unsuccessful